contrib/systemd/cilium-consul.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=cilium-consul
+Documentation=https://github.com/cilium/cilium
+Requires=docker.service
```

Comment:
Long term we should probably have a cilium-consul meta unit which can be implemented by either cilium-consul-docker or cilium-consul-rkt. Not sure what the systemd native way is to do this.